include/dsn/cpp/serialization_helper/dsn.layer2_types.h

```diff
@@ -276,6 +278,58 @@ inline std::ostream &operator<<(std::ostream &out, const configuration_query_by_
     return out;
 }
 
+typedef struct _duplication_options__isset
```

Comment:
As your new commit doesn't update `dsn.layer2.thrift` files, please discard the update of `dsn.layer2_types.h` and `dsn.layer2_types.cpp`

Reply:
done